components/schema-migrator/migrations/kyma-environment-broker/202009171000_add_runtime_states.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS runtime_states (
```

Comment:
every time we query for the runtime state we have runtime ID, so let's create an index for that